creme/sketch/bricks.py

```diff
@@ -59,14 +60,14 @@ class BarChartBrick(ChartBrick):
     """
     template_name = "sketch/bricks/bar-chart.html"
 
-    # Draw limit lines on the Y axis
-    limits = []
+    # Show limits along the Y axis : list of values drawn as horizontal lines
+    limits: Iterable[Union[int, float]] = ()
```

Comment:
```
from __future__ import annotations
[...]
    limits: Sequence[int | float] = ()
```

(I prefer sequence here because it's more easier to manipulate them in a hack -- e.g  `limits[1:3] + ...`)